build/flutter_app.gni

```diff
@@ -71,7 +71,6 @@ template("flutter_app") {
     ]
 
     outputs = [
-      bundle_path,
```

Comment:
Why shouldn't the bundle_path be listed as an output?  It's ultimately the thing this rule is trying to create.